### PR TITLE
Increase precision of initialBearing/finalBearing...

### PR DESCRIPTION
**Pull request by Surfoo**
from @olivierstern, with unit tests fixed. see #124

### CODE REVIEW

**Comment by Surfoo** [comment]
ping @Nyholm :)

File: tests/Vertex/VertexTest.php [hunk 1]
```diff
@@ -175,7 +175,7 @@ public function coordinatesAndExpectedInitialCardinalProvider()
             array(
                 array('28.8234055', '1.3072664'),
                 array('43.296482', '5.36978'),
-                'N'
+                'NNE'
```
Comment (Nyholm) [inline tests/Vertex/VertexTest.php:178]: Is this change related?

**Review by Nyholm — APPROVED** [review]
Thank you

File: tests/Vertex/VertexTest.php (hunk 1 above)
Comment (Surfoo) [reply tests/Vertex/VertexTest.php:178]: Yes, the direction changed a little bit for this test.
